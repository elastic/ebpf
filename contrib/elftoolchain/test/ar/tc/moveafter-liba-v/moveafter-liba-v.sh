# $Id$
inittest moveafter-liba-v tc/moveafter-liba-v
extshar ${TESTDIR}
extshar ${RLTDIR}
runcmd "${AR} mav a2.o liba.a a3.o a4.o" work true
runcmd "plugin/teraser -ce -t moveafter-liba-v liba.a" work false
runcmd "plugin/teraser -e liba.a" result false
rundiff true
