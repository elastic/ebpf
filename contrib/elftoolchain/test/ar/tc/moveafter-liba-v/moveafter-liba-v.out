m - a3.o
m - a4.o
